pkg/webui/console/containers/lora-cloud-gls-form/index.js

```diff
@@ -191,95 +192,113 @@ const LoRaCloudGLSForm = () => {
     [setMultiFrame, formRef],
   )
 
+  const [automaticMultiFrame, setAutomaticMultiFrame] = useState(
+    initialValues.data.multi_frame_window_size === 0 ||
+      initialValues.data.multi_frame_window_size === undefined,
+  )
+  const handleAutomaticWindowSizeChange = useCallback(
+    evt => {
+      const checked = evt.target.checked
+      setAutomaticMultiFrame(checked)
+    },
+    [setAutomaticMultiFrame],
+  )
+
   useEffect(() => {
     setQueryType(initialValues.data.query)
     setMultiFrame(initialValues.data.multi_frame)
   }, [initialValues.data.query, initialValues.data.multi_frame])
 
   return (
-    <RequireRequest
-      requestAction={getAppPkgDefaultAssoc(appId, LORA_CLOUD_GLS.DEFAULT_PORT, selector)}
+    <Form
+      error={error}
+      validationSchema={validationSchema}
+      initialValues={initialValues}
+      onSubmit={handleSubmit}
+      formikRef={formRef}
     >
-      <Form
-        error={error}
-        validationSchema={validationSchema}
-        initialValues={initialValues}
-        onSubmit={handleSubmit}
-        formikRef={formRef}
-      >
-        <Form.Field
-          component={Input}
-          title={sharedMessages.token}
-          description={m.tokenDescription}
-          name="data.token"
-          required
-        />
-        <Form.Field
-          component={Select}
-          title={m.queryType}
-          description={m.queryTypeDescription}
-          name="data.query"
-          options={LORACLOUD_GLS_QUERY_LABELS}
-          disabled={LORACLOUD_GLS_QUERY_LABELS.length === 1}
-          onChange={handleQueryTypeChange}
-          required
-        />
-        {queryType === LORACLOUD_GLS_QUERY_TYPES.TOARSSI && (
-          <>
-            <Form.Field
-              component={Checkbox}
-              title={m.multiFrame}
-              description={m.multiFrameDescription}
-              name="data.multi_frame"
-              onChange={handleMultiFrameChange}
-              required
-            />
-            {multiFrame && (
-              <>
+      <Form.Field
+        component={Input}
+        title={sharedMessages.token}
+        description={m.tokenDescription}
+        name="data.token"
+        required
```

Comment:
When submitting the form without `token` there is no error rendered under this field.